server/config.js

```diff
@@ -0,0 +1,4 @@
+module.exports = {
+  DB: 'postgres://sfrhrrwr:bMdc_tEbhDhPTFRZkHZ8nHYDgvFHmbcP@pellefant.db.elephantsql.com:5432/sfrhrrwr',
```

Comment:
You've got your credentials listed here on line 2. I don't recall if Charlie directed you to do that, but probably best to deactivate this DB at this point. I doubt anyone would get in there and mess with it, but someone could potentially get in here with your creds and add/edit/delete your DB tables.

Depending on what Elephant SQL will allow, it could potentially allow someone to add a ton of rows, putting you into a paid tier without your knowing (causing unnecessary costs).